test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SamplesEndToEndTests_PowerShell.cs

```diff
@@ -51,7 +51,7 @@ static TestFixture()
             }
 
             public TestFixture()
-                : base(Path.Combine(Environment.CurrentDirectory, @"..", "..", "..", "..", "..", "sample", "PowerShell"), "samples", RpcWorkerConstants.PowerShellLanguageWorkerName)
+                : base(Path.Combine(Environment.CurrentDirectory, @"..", "..", "..", "..", "..", "sample", "PowerShell"), "samples", RpcWorkerConstants.PowerShellLanguageWorkerName, functionsWorkerRuntimeVersion: "7")
```

Comment:
@AnatoliB @Francisco-Gamino FYI, I'm updating the samples E2E test to use PS 7 moving forward. Wanted to make sure you were aware. This removes the dependency on .NET 2.1